pkgs/stdenv/generic/setup.sh

```diff
@@ -141,11 +141,11 @@ trap "exitHandler" EXIT
 
 
 addToSearchPathWithCustomDelimiter() {
-    local delimiter=$1
-    local varName=$2
-    local dir=$3
+    local delimiter="$1"
+    local varName="$2"
+    local dir="$3"
     if [ -d "$dir" ]; then
-        eval export ${varName}=${!varName}${!varName:+$delimiter}${dir}
+        export "${varName}=${!varName}${!varName:+$delimiter}${dir}"
```

Comment:
Oh. I had this branch sitting around for a while before my current tail-between-legs conservationism :D. But, spare me this one eval removal if stdenv builds :D:D?